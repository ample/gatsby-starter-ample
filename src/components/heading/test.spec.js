import React from "react"
import renderer from "react-test-renderer"

import { component as Heading, fixtures } from "."

describe("Heading", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Heading {...fixtures.h1} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
