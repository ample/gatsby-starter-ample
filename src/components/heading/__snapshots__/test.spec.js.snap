// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading renders correctly 1`] = `
<h1
  className="heading_level_h1"
  dangerouslySetInnerHTML={
    Object {
      "__html": "Lorem ipsum dolor sit amet",
    }
  }
/>
`;
